napari/_vispy/layers/image.py

```diff
@@ -169,11 +169,22 @@ def _on_depiction_change(self):
     def _on_colormap_change(self):
         self.node.cmap = VispyColormap(*self.layer.colormap)
 
+    def _update_mip_minip_cutoff(self):
+        if isinstance(self.node, VolumeNode):
+            if self.layer.blending != 'opaque':
```

Comment:
shoudl this check if the mode is translucent, rather than not opaque? for example, do we want the empty background of an image to be blended additively, or simply be ignored?

Reply:
I would expect the blending to happen on top of a background in additive/subtractive
```suggestion
            if self.layer.blending == 'translucent':
```

Reply:
Agreed, but this should also include `translucent_no_depth` I suppose.